src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -514,6 +514,7 @@ export type SentryAppWebhookRequest = {
     name: string;
   };
   responseCode: number;
+  errorUrl?: string;
```

Comment:
@nolachen is the PR to show the link coming later?

Reply:
@scefali Yeah this is just setting everything up so it's possible